frontend/public/module/k8s/pods.ts

```diff
@@ -69,6 +69,11 @@ export const VolumeSource = {
     label: 'Gluster FS',
     description: 'GlusterFS volume that will be mounted on the host machine.',
   },
+  persistentVolumeClaim: {
```

Comment:
I'm not sure this is right. Is this to change the mounted volumes table? I think this will conflict with #1344.

Reply:
Yes, PR1344 does this already.  I only added for testing so I will not make it a part of this PR.